app/api/upload.py

```
@@ -26,6 +36,217 @@
+                                await asyncio.sleep(2 ** retry_count)  # 指数バックオフ
+                            else:
+                                logger.error(f"Max retries reached for {job_id}, skipping")
+        else:
+            local_dir = os.path.join(settings.get_session_dirpath(session_id), "pdfs")
+            if not os.path.exists(local_dir):
+                logger.info(f"Creating local directory: {local_dir}")
+                os.makedirs(local_dir, exist_ok=True)
+                
+            logger.info(f"Checking for PDF files in local directory: {local_dir}")
+            if os.path.exists(local_dir) and os.listdir(local_dir):
+                for job_id in job_ids:
```

Ruff (0.11.9)

Loop control variable `job_id` not used within loop body

(B007)